### PR TITLE
Updated the RisingStack.com links and added a link to Trace

### PR DESCRIPTION
**Pull request by hellogabor**

### CODE REVIEW

File: partials/sidebar-external.hbs [hunk 1]
```diff
@@ -1,11 +1,12 @@
-<li class="external-link"><a target="_blank" href="http://risingstack.com"><i class="fa fa-globe"></i> RisingStack Website</a></li>
+<li class="external-link"><a target="_blank" href="https://risingstack.com"><i class="fa fa-globe"></i> RisingStack Website</a></li>
+<li class="external-link"><a target="_blank" href="http://trace.risingstack.com"><i class="fa fa-bullseye"></i> Topology view for microservices</a><a class="new"> New!</a></li>
 <li class="external-link"><a target="_blank" href="https://github.com/risingstack"><i class="fa fa-github"></i> GitHub</a></li>
 <li class="external-link"><a target="_blank" href="https://twitter.com/risingstack"><i class="fa fa-twitter"></i> Twitter</a></li>
-<li class="external-link"><a target="_blank" href="mailto:info@risingstack.com"><i class="fa fa-envelope"></i> Contact</a></li>
+<li class="external-link"><a target="_blank" href="mailto:hello@risingstack.com"><i class="fa fa-envelope"></i> Contact</a></li>
```
Comment (gergelyke) [inline partials/sidebar-external.hbs]: why changing it?

Comment (hellogabor) [inline partials/sidebar-external.hbs]: To see if someone contacts us so I can track the leads.

Comment (gergelyke) [inline partials/sidebar-external.hbs]: better idea: how about funneling it into the contact form?

Comment (hellogabor) [inline partials/sidebar-external.hbs]: cool! on it

**Pull request merged**